app/Http/Controllers/CategoryController.php

```diff
@@ -1,111 +0,0 @@
-<?php
-
-namespace App\Http\Controllers;
-
-use Illuminate\Http\Request;
-use App\Category;
-
-class CategoryController extends Controller
-{
-
-    public function __construct()
-    {
-        $this->middleware('auth');
-    }
-
-    /**
-     * Display a listing of the resource.
-     *
-     * @return \Illuminate\Http\Response
-     */
-    public function index()
-    {
-        $this->authorize('viewAny', Category::class);
-        // display a view of all of our categories
-        $categories = Category::all();
-        return view('categories.index', compact('categories'));
-    }
-
-    /**
-     * Store a newly created resource in storage.
-     *
-     * @param  \Illuminate\Http\Request  $request
-     * @return \Illuminate\Http\Response
-     */
-    public function store(Request $request)
-    {
-        $this->authorize('viewAny', Category::class);
```

Comment:
Change the authorize to use the respective (create(?)) policy method. Goes for every mehtod in this (and other) class(es)